debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlJdbcContext.java

```diff
@@ -48,7 +48,7 @@
     protected static ConnectionFactory FACTORY = JdbcConnection.patternBasedFactory(MYSQL_CONNECTION_URL,
             JdbcConfiguration.PORT.withDefault(MySqlConnectorConfig.PORT.defaultValueAsString()));
 
-    protected final Logger logger = LoggerFactory.getLogger(getClass());
+    protected static final Logger logger = LoggerFactory.getLogger(MySqlJdbcContext.class);
```

Comment:
@gunnarmorling  The lowercase was left intentionally to not inflate the PR. This class does not exist in new impl so it is not an important detail.

Reply:
Agreed, makes sense!